src/server.ts

```diff
@@ -276,25 +276,33 @@ async function announce(
 		res.end(e.message);
 	}
 }
-
+async function statusCheck(
+	req: IncomingMessage,
+	res: ServerResponse,
+): Promise<void> {
+	res.writeHead(200);
+	res.end("OK");
+}
 async function handleRequest(
 	req: IncomingMessage,
 	res: ServerResponse,
 ): Promise<void> {
 	if (!(await authorize(req, res))) return;
+	const endpoint = req.url!.split("?")[0];
 
-	if (req.method !== "POST") {
+	if (req.method !== "POST" && endpoint !== "/api/status") {
```

Comment:
this leaves the door open for /api/status to be called with either POST or GET. we should pick one and enforce it

Reply:
I was aware of this when writing it, but it would obviously be a GET in this case, and I assumed no one would be POST'ing to it, and even if so it would still return 200 OK.

I'll add GET exclusivity.